### PR TITLE
Move functionality to cogs to tidy the place up a bit

### PR DESCRIPTION
**Pull request by LachlanCourt**
Much neater now :)

### CODE REVIEW

File: cogs/GlobalConfig.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import sys, json
+
+class GlobalConfig():
+    def __init__(self):
```
Comment (Y0sh1dk) [inline cogs/GlobalConfig.py:4]: I dont think we should be parsing the files in the default constructor, I think it should just setup the class variables and then have a overloaded constructor doing something like
```
def __init__(self, configPath, roleDataPath, lockedDataPath)
    self()
    # parse files here
```

**Review by Y0sh1dk — CHANGES_REQUESTED** [review]

File: cogs/GlobalConfig.py (hunk 1 above)
Comment (LachlanCourt) [reply cogs/GlobalConfig.py:4]: Just keep it as storage? Aight

**Review by Y0sh1dk — APPROVED** [review]
Nice!, assuming it works, LGTM. Much more maintainable now.

File: cogs/GlobalConfig.py (hunk 1 above)
Comment (Y0sh1dk) [reply cogs/GlobalConfig.py]: I will refactor this slightly in my arg parse PR.

**Pull request merged**